bin/tsdfx/copy.c

```diff
@@ -273,6 +292,26 @@ tsdfx_copy_delete(struct tsd_task *t)
 }
 
 /*
+ * Remove old source files.
+ */
+static void
+tsdfx_copy_purgesource_child(void *ud)
+{
+	struct tsdfx_copy_task_data *ctd = ud;
+
+	/*
+	 * At this point, running as the owner of the file.
+	 */
+	if (-1 == remove(ctd->src)) {
+		/*
+		 * FIXME: this message do not make it into the log.
```

Comment:
The task doesn't have `TASK_STDERR_PIPE` set, so the child's `stderr` goes to wherever the parent's goes (`/dev/null` when daemonized). You need to set `TASK_STDERR_PIPE` and have the parent read the output like we do for scan tasks, or call either `tsdfx_log_init()` or `tsdfx_log_userlog()` and use the appropriate macro.